pymc_experimental/distributions/continuous.py

```diff
@@ -280,3 +280,109 @@ def __new__(cls, name, nu, **kwargs):
     @classmethod
     def dist(cls, nu, **kwargs):
         return CustomDist.dist(nu, dist=cls.chi_dist, class_name="Chi", **kwargs)
+
+
+class Maxwell:
+    R"""
+    The Maxwell-Boltzmann distribution
+
+    The pdf of this distribution is
+
+    .. math::
+
+       f(x \mid a) = {\displaystyle {\sqrt {\frac {2}{\pi }}}\,{\frac {x^{2}}{a^{3}}}\,\exp \left({\frac {-x^{2}}{2a^{2}}}\right)}
+
+    Read more about it on `Wikipedia <https://en.wikipedia.org/wiki/Maxwell%E2%80%93Boltzmann_distribution>`_
+
+    .. plot::
+        :context: close-figs
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+        import scipy.stats as st
+        import arviz as az
+        plt.style.use('arviz-darkgrid')
+        x = np.linspace(0, 20, 200)
+        for a in [1, 2, 5]:
+            pdf = st.maxwell.pdf(x, scale=a)
+            plt.plot(x, pdf, label=r'$a$ = {}'.format(a))
+        plt.xlabel('x', fontsize=12)
+        plt.ylabel('f(x)', fontsize=12)
+        plt.legend(loc=1)
+        plt.show()
+
+    ========  =========================================================================
+    Support   :math:`x \in (0, \infty)`
+    Mean      :math:`2a \sqrt{\frac{2}{\pi}}`
+    Variance  :math:`\frac{a^2(3 \pi - 8)}{\pi}`
+    ========  =========================================================================
+
+    Parameters
+    ----------
+    a : tensor_like of float
+        Scale parameter (a > 0).
+
+    """
+
+    @staticmethod
+    def maxwell_dist(a: TensorVariable, size: TensorVariable) -> TensorVariable:
+        return Chi.dist(nu=3, size=size) * a
```

Comment:
This will be wrong when `size` is not provided (there is a utility for that in pymc shape_utils) and `a` is a vector. We need to check for that case and do `size=a.shape` when it happens.

Reply:
make sense. I am taking a look at the available functions [here](https://www.pymc.io/projects/docs/en/stable/_modules/pymc/distributions/shape_utils.html)

Reply:
would this have to be implemented for Chi as well? [here](https://github.com/pymc-devs/pymc-experimental/blob/40e28e85516785e26cabfb9051bbc487793a399d/pymc_experimental/distributions/continuous.py#L272-L273) or is it only a concern with a multiplication?

Reply:
I don't think Chi has any argument that goes outside a dist?

Reply:
yeah, makes sense

Reply:
Here we need something like `if rv_size_is_none(a): size = a.shape`

Reply:
added. Didn't mean to close before making changes 😁